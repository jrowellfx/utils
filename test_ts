./ts -x
./ts --xxx
./ts -h
./ts --help
./ts 
./ts --short
./ts --fullyear
./ts --fullyear --short
