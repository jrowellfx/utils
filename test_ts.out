Usage: ts [-h | --help] [OPTION]...
Usage: ts [-h | --help] [OPTION]...
Usage: ts [-h | --help] [OPTION]...

ts prints out a string suitable for a timestamp, as YYMMDD-hhmmss

optional arguments:

 -h, --help     show this help and exit
 --onlyDate     skip printing the time, only print the date
 --short        same as --onlyDate
 --fullYear     print year as YYYY instead of just YY
 --noSeconds    print time as hhmm only
 --noMinutes    print time as hh only
 --onlyTime     skip printing the date, only print the time
 --roundQuarter round minutes down to the nearest 15 
 --roundHalf    round minutes down to the nearst half hour
 --version      print out the version number and exit

Usage: ts [-h | --help] [OPTION]...

ts prints out a string suitable for a timestamp, as YYMMDD-hhmmss

optional arguments:

 -h, --help     show this help and exit
 --onlyDate     skip printing the time, only print the date
 --short        same as --onlyDate
 --fullYear     print year as YYYY instead of just YY
 --noSeconds    print time as hhmm only
 --noMinutes    print time as hh only
 --onlyTime     skip printing the date, only print the time
 --roundQuarter round minutes down to the nearest 15 
 --roundHalf    round minutes down to the nearst half hour
 --version      print out the version number and exit

Tue Apr 16 13:22:30 CDT 2024
240416-132230
240416-1322
240416-13
240416-1315
240416-1300
132230
1315
1300
240416
240416
20240416-132230
20240416-1315
20240416-1300
20240416
20240416-1322
20240416-13
1.2.0
