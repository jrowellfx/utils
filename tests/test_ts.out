./ts: no such option -x
usage: ts [-h | --help] [OPTION]...
./ts: no such option --xxx
usage: ts [-h | --help] [OPTION]...
usage: ts [-h | --help] [OPTION]...

ts prints out a string suitable for a timestamp, as YYMMDD-hhmmss

optional arguments:

 -h, --help      show this help and exit
 --only-date     skip printing the time, only print the date
 --short         same as --only-date
 --full-year     print year as YYYY instead of just YY
 --no-seconds    print time as hhmm only
 --no-minutes    print time as hh only
 --only-time     skip printing the date, only print the time
 --round-quarter round minutes down to the nearest 15 
 --round-half    round minutes down to the nearst half hour
 --version       print out the version number and exit

usage: ts [-h | --help] [OPTION]...

ts prints out a string suitable for a timestamp, as YYMMDD-hhmmss

optional arguments:

 -h, --help      show this help and exit
 --only-date     skip printing the time, only print the date
 --short         same as --only-date
 --full-year     print year as YYYY instead of just YY
 --no-seconds    print time as hhmm only
 --no-minutes    print time as hh only
 --only-time     skip printing the date, only print the time
 --round-quarter round minutes down to the nearest 15 
 --round-half    round minutes down to the nearst half hour
 --version       print out the version number and exit

Sun Feb  2 17:16:41 CST 2025
250202-171641
250202-1716
250202-17
250202-1715
250202-1700
171641
1715
1700
250202
250202
20250202-171641
20250202-1715
20250202-1700
20250202
20250202-1716
20250202-17
2.0.0
